### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by Eukst**
features added : 
Added Static Pages 
Added bootstrap 
Fix flash message error of Sign In , Sign Out, Sign Up

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
```
Comment (Sharee-ongraph) [inline app/controllers/static_pages_controller.rb:4]: add comments for class and correct indendation for empty methods

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  root 'static_pages#home'
+  get 'static_pages/help'
+  get 'static_pages/contact'
+  get 'static_pages/about'
   devise_for :users
-  root "application#index"
+  # root "application#index"
```
Comment (mitaligaur) [inline config/routes.rb]: Remove if not required

**Review by mitaligaur — CHANGES_REQUESTED** [review]

File: app/controllers/static_pages_controller.rb (hunk 1 above)
Comment (Eukst) [reply app/controllers/static_pages_controller.rb:4]: done

**Review by Sharee-ongraph — APPROVED** [review]

**Review by mitaligaur — APPROVED** [review]

File: config/routes.rb (hunk 2 above)
Comment (Eukst) [reply config/routes.rb]: done

**Pull request merged**